### ISSUE TITLE
crypto-square: check test version and update if necessary

### ISSUE THREAD

**Issue by N-Parsons** [exercism/python#994]
As identified in #784, some of the tests currently don't reference the version number for the `canonical-data.json` that they are based on, `crypto-square` is one of them. In order to help with the ongoing maintenance of this track, we're in the process of checking the tests against the current version of the `canonical-data.json` - if they are up-to-date, then you can just add the version string, but if they are out-of-date, then the tests will need updating.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v3.1.0
```

**Test suite:** [exercism/python/exercises/crypto-square/crypto_square_test.py](https://github.com/exercism/python/blob/master/exercises/crypto-square/crypto_square_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/crypto-square/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/canonical-data.json)

**Comment by elyssonmr** [comment]
Going to do this issue :D

**Comment by cmccandless** [comment]
Fixed in #1051
